acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/web3j/privacy/PrivCallAcceptanceTest.java

```diff
@@ -0,0 +1,244 @@
+
+import org.hyperledger.besu.tests.acceptance.dsl.privacy.PrivacyAcceptanceTestBase;
+import org.hyperledger.besu.tests.acceptance.dsl.privacy.PrivacyNode;
+import org.hyperledger.besu.tests.web3j.generated.EventEmitter;
+
+import java.io.IOException;
+import java.math.BigInteger;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.jetbrains.annotations.NotNull;
+import org.junit.Before;
+import org.junit.Test;
+import org.web3j.abi.FunctionEncoder;
+import org.web3j.abi.TypeReference;
+import org.web3j.abi.datatypes.Function;
+import org.web3j.abi.datatypes.Type;
+import org.web3j.abi.datatypes.generated.Uint256;
+import org.web3j.protocol.core.Request;
+import org.web3j.protocol.core.methods.request.Transaction;
+import org.web3j.protocol.core.methods.response.EthCall;
+import org.web3j.protocol.core.methods.response.TransactionReceipt;
+import org.web3j.protocol.exceptions.ClientConnectionException;
+import org.web3j.protocol.http.HttpService;
+import org.web3j.tx.Contract;
+
+public class PrivCallAcceptanceTest extends PrivacyAcceptanceTestBase {
+
+  private static final long POW_CHAIN_ID = 2018;
+  private static final int VALUE = 1024;
+
+  private PrivacyNode minerNode;
+
+  @Before
+  public void setUp() throws Exception {
+    minerNode =
+        privacyBesu.createPrivateTransactionEnabledMinerNode(
+            "miner-node", privacyAccountResolver.resolve(0));
+    privacyCluster.start(minerNode);
+  }
+
+  @Test
+  public void privCallMustReturnCorrtectValue() throws Exception {
+
+    final String privacyGroupId =
+        minerNode.execute(
+            privacyTransactions.createPrivacyGroup(
+                "myGroupName", "my group description", minerNode));
+
+    final EventEmitter eventEmitter =
+        minerNode.execute(
+            privateContractTransactions.createSmartContractWithPrivacyGroupId(
+                EventEmitter.class,
+                minerNode.getTransactionSigningKey(),
+                POW_CHAIN_ID,
+                minerNode.getEnclaveKey(),
+                privacyGroupId));
+
+    privateContractVerifier
+        .validPrivateContractDeployed(
+            eventEmitter.getContractAddress(), minerNode.getAddress().toString())
+        .verify(eventEmitter);
+
+    final Request<Object, EthCall> priv_call = privCall(privacyGroupId, eventEmitter, false, false);
+
+    EthCall resp = priv_call.send();
+
+    String value = resp.getValue();
+    assertThat(new BigInteger(value.substring(2), 16)).isEqualByComparingTo(BigInteger.ZERO);
+
+    final TransactionReceipt receipt = eventEmitter.store(BigInteger.valueOf(VALUE)).send();
+    assertThat(receipt).isNotNull();
+
+    resp = priv_call.send();
+    value = resp.getValue();
+    assertThat(new BigInteger(value.substring(2), 16))
+        .isEqualByComparingTo(BigInteger.valueOf(VALUE));
+  }
+
+  @Test
+  public void privCallWithNonExistingPrivacyGroupMustNotSucceed() {
```

Comment:
`privCallWithoutExistingPrivacyGroupMustException`
or
`noExistingPrivacyGroupMustException`
(as the class has the prefix `PrivCall`, also having as a method prefix seems redundant).

Reply:
Fixed the test method names